content/css/concepts/filter-functions/terms/saturate/saturate.md

```diff
@@ -24,17 +24,24 @@ filter: saturate(<value>);
 
 where a required `<value>` can be one of the following:
 
-- Number value: `0`, `1.5`
-- Percentage value: `100%`, `50%`
+- Number value: `0`, `0.5`, `1.0`, `1.5`
+- Percentage value: `0%`, `50%`, `100%`, `150%`
```

Comment:
Can you recheck this once, because I guess the `saturate()` can take any value and is not limited to these 4 values only?

Reply:
Thankyou for the feedback! I've revised the explanation for the `saturate()` value. Please check it out, and let me know if there's anything else.